### DISCUSSION TITLE
corona-warn-app.opensource@sap.com still displays on https://github.com/corona-warn-app

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3545] [Development] [answered]
The e-mail address corona-warn-app.opensource@sap.com still displays on https://github.com/corona-warn-app which is the GitHub organization site for `corona-warn-app`.

![image](https://github.com/corona-warn-app/cwa-website/assets/66998419/bcb01a7f-2f40-4e24-97ac-5465e139df3c)

This should be replaced by CoronaWarnApp@rki.de.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks, I will forward this!

**Reply by MikeMcC399** [reply, marked as answer]
Fixed
